x-pack/plugins/secops/public/components/timeline/body/column_headers/headers.ts

```diff
@@ -35,9 +35,9 @@ export const defaultHeaders: ColumnHeader[] = [
   },
   {
     columnHeaderType: 'not-filtered',
-    id: 'event.type',
+    id: 'event.action',
```

Comment:
Thanks for addressing https://github.com/elastic/ingest-dev/issues/291 with this!